.ko.yaml

```diff
@@ -0,0 +1,15 @@
+builds:
+  - id: anchore-admission-controller
+    main: ./cmd/kubernetes-admission-controller
+    env:
+      - CGO_ENABLED=0
+    flags:
+      - -tags
+      - -trimpath
```

Comment:
AFAIK these are unneeded. `ko` already builds with `CGO_ENABLED=0` and `-trimpath` by default, and I think `-tags` without any values is a no-op.

Reply:
ack! Thanks!